### ISSUE TITLE
Main landmark is not provided to the ONNX Runtime landing page: A11y_WCP URLs - ONNX Runtime_ONNX Runtime_Home_AI4W.

### ISSUE THREAD

**Issue by PadminiGariki** [microsoft/onnxruntime#20100]
### GitHub Tags:
#A11yWCAG2.1; #A11ySev2; #A11yTCS; #DesktopWeb; #SH_ONNX Runtime & Ecosystem_Web_Mar2024; #ONNX Runtime & Ecosystem; #WCAG1.3.1; #WCP; #Win11; #FTP; #P3_WCP; #Element:Landmarks; #ONNX Runtime; #Info and Relationships; #ChromiumEdge; #A11yMAS;

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver".** 

### Environment Details:
Application Name: WCP URLs - ONNX Runtime
URl: https://onnxruntime.ai/
Edge version: 123.0.2420.53 (Official build) (64-bit)
OS: Windows 11 version 23H2

### Steps to Reproduce

1. Hit the URL "https://onnxruntime.ai/".
2. "ONNX Runtime" home page is appeared.
3. Turn on Landmark toggle in accessibility Insights for web.
4. Now check that main landmark is not provided to the ONNX Runtime landing page.

### Actual Results:
Main landmark is not provided to the ONNX Runtime landing page.

### Expected Results:
The expected behavior is that all pages should include landmarks to help users navigate the page. Landmarks should be clearly labeled and easy to identify and main landmark should be provided for a page.

### User Impact:
People with cognitive and motor disabilities may have difficulty navigating the page without landmarks. Without landmarks, they may not be able to find the content they are looking for or understand the page structure.

### Attachment:

![Screenshot 2024-03-27 144259](https://github.com/microsoft/onnxruntime/assets/142479541/679cbe53-ed54-4f57-87ab-46ad97d5e8e4)

**Comment by PadminiGariki** [comment]
Verified the issue in below environment.
Application Name: WCP URLs - ONNX Runtime
URl: https://onnxruntime.ai/
Edge version: 123.0.2420.97 (Official build) (64-bit)
OS: Windows 11 version 23H2

Issue is fixed. Hence the bug is closed.
#Regressed:04-18-24; #Closed;

For reference attachment is added.

![Screenshot 2024-04-18 114310](https://github.com/microsoft/onnxruntime/assets/142479541/1e1c57df-afdc-4718-accd-5b4eb0eb67f6)
